.github/workflows/release.yml

```diff
@@ -39,7 +36,7 @@ jobs:
           PASSPHRASE: ${{ secrets.PASSPHRASE }}
       -
         name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v2.7.0
+        uses: goreleaser/goreleaser-action@v2
```

Comment:
Major version should be fine here. If you wanna lock to a specific goreleaser, do that instead.